src/main/java/domain/PurchaseAmount.java

```diff
@@ -0,0 +1,40 @@
+package domain;
+
+import exception.AppException;
+
+public class PurchaseAmount {
+
+    public static final String INVALID_PURCHASEAMOUNT_UNIT = "구입 금액은 1000 단위 숫자로 입력해야 합니다.";
+    public static final String INVALID_PURCHASEAMOUNT_RANGE = "구입 금액은 1000원 이상이어야 합니다.";
+    public static final String INVALID_PURCHASEAMOUNT_FORMAT = "구입 금액은 숫자여야 합니다.";
+
+    private static final int UNIT = 1_000;
```

Comment:
아래와 같이 하면 로또 가격이 변경될 때 수정할 부분이 훨씬 줄어들겠군요!

```suggestion
    private static final int UNIT = 1_000;
    public static final String INVALID_PURCHASEAMOUNT_UNIT = String.format("구입 금액은 %d 단위 숫자로 입력해야 합니다.", UNIT);
    public static final String INVALID_PURCHASEAMOUNT_RANGE = String.format("구입 금액은 %d원 이상이어야 합니다.", UNIT);
    public static final String INVALID_PURCHASEAMOUNT_FORMAT = "구입 금액은 숫자여야 합니다.";
```